aodncore/pipeline/templates/notify.html.j2

```diff
@@ -42,8 +42,20 @@
     </div>
     {% endif %}
 
+    <p>
+    For further information about this message, please contact <a href="mailto:info@aodn.org.au">info@aodn.org.au</a>
+    (or the relevant AODN project officer).
+    </p>
+
     <p>Regards,<br>
         Australian Ocean Data Network
     </p>
+
+    <footer>
+    <pre>
+    (Log reference: {{ celery_task_name }}[{{ celery_task_id }}])
```

Comment:
I'd just use a "p" tag for this, since then it could be styled to be smaller than the body text, to de-emphasise it a bit. e.g.
```css
    .fineprint {
        font-size: 10px;
    }
```
```html
    <footer>
        <p class="fineprint">
            (Reference: {{ celery_task_name }}[{{ celery_task_id }}])
        </p>
    </footer>
```